sound/soc/sof/ipc.c

```diff
@@ -483,7 +495,7 @@ static void ipc_xrun(struct snd_sof_dev *sdev, u32 msg_id)
 	struct snd_sof_pcm_stream *stream;
 	struct sof_ipc_stream_posn posn;
 	struct snd_sof_pcm *spcm;
-	int direction;
+	int direction, ret;
```

Comment:
maybe these to separate lines... I've been always told about that :)... otherwise LGTM

Reply:
same type can be in the same line except for the ones which is initialized.

Reply:
@juimonen at least I can with a high certainty suggest, that it wasn't me who asked you to do that ;-) If you look through our sources they contain both. I personally usually prefer grouping variables. One of the reasons is the same why I prefer to avoid duplicate or otherwise superfluous empty lines. I prefer to see as much on my screen without paging as possible. My first Pascal program was written column 0 to column 79 over pages and pages of text :-D No, let's not go there, but `int i; int j; int ret; int err; int i_just_need_one_more_counter;` each on a separate line is an overkill too IMHO